android/src/newarch/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -420,39 +420,46 @@ public void setUserAgent(RNCWebViewWrapper view, @Nullable String value) {
         mRNCWebViewManagerImpl.setUserAgent(view, value);
     }
 
-    // These will never be called because we use the shared impl for now
   @Override
   public void goBack(RNCWebViewWrapper view) {
-    view.getWebView().goBack();
+    // NOTE: It is handled via receiveCommand().
+    // view.getWebView().goBack();
   }
 
   @Override
   public void goForward(RNCWebViewWrapper view) {
-    view.getWebView().goForward();
+    // NOTE: It is handled via receiveCommand().
+    // view.getWebView().goForward();
   }
 
   @Override
   public void reload(RNCWebViewWrapper view) {
-    view.getWebView().reload();
+    // NOTE: It is handled via receiveCommand().
+    // view.getWebView().reload();
   }
 
   @Override
   public void stopLoading(RNCWebViewWrapper view) {
-    view.getWebView().stopLoading();
+    // NOTE: It is handled via receiveCommand().
+    // view.getWebView().stopLoading();
   }
 
   @Override
   public void injectJavaScript(RNCWebViewWrapper view, String javascript) {
-      view.getWebView().evaluateJavascriptWithFallback(javascript);
+    // NOTE: It is handled via receiveCommand().
+    // view.getWebView().evaluateJavascriptWithFallback(javascript);
   }
 
   @Override
   public void requestFocus(RNCWebViewWrapper view) {
-      view.requestFocus();
+    // NOTE: It is handled via receiveCommand().
+    // view.requestFocus();
```

Comment:
Why is the purpose of leaving empty overrides?

Reply:
@o-alexandrov I don't have previous experience with native UI components, and I haven't looked much into how exactly that `codegenNativeCommands()` and `.receiveCommands()` stuff works. Just looking at the existing code and behavior it looked to me that if these `.requestFocus()`, _etc._ methods get called directly, perhaps they are the best way to implement related functionality (and probably they are abstract in a parent class, thus have to be overridden anyway &mdash; but I haven't really checked it). Then, there is the backward compatibility to the old arch, I thought if I touch `.receiveCommands()`, perhaps it is needed for the old arch, and it will need a bunch more efforts to ensure it works there.

My immediate interest here was just to hotfix the bug causing me troubles, thus I figured out this is the minimal intrusion way to do it &mdash; comment out the calls that cause duplicated command executions, but otherwise leave these method in the codebase so that somebody later looks second time into it.